Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -99,6 +100,8 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._appendStackTrace();
 
         this._renderRepeatCount();
+        if (WI.settings.showTimestamps.value)
```

Comment:
Should `renderTimestamp` check this setting itself so it can remove the element if necessary?